### PR TITLE
Add ec2:DescribeDhcpOptions permissions to capa controller

### PR DESCRIPTION
**Pull request by fiunchinho**
CAPA 2.4.x requires this new permission.

## Checklist

- [X] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Add `ec2:ReplaceRoute` permissions to the CAPA controller role
+### Added
+
+- Add `ec2:ReplaceRoute` permissions to the CAPA controller role.
```
Comment (T-Kukawka) [inline CHANGELOG.md:12]: can we add the note that new CAPA version requires it?

Comment (fiunchinho) [reply CHANGELOG.md:12]: Added note about it, PTAL.

Comment (T-Kukawka) [reply CHANGELOG.md:12]: why only in 1 bullet point not the other? :D

Comment (fiunchinho) [reply CHANGELOG.md:12]: That was added in another PR. I'll try to figure out why is needed.

**Review by T-Kukawka — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fiunchinho) [reply CHANGELOG.md:12]: oh you approved and the branch got removed :sweat_smile:

**Pull request merged**